Secure-Software-Development-Policy/Zowe-SSDP-SDLC/Best-Practices/Document/license-copyright.md

```diff
@@ -0,0 +1,41 @@
+# Considerations for Coding Contributions to Zowe
```

Comment:
The ENTIRE? folder of best practices appears to come from what we already have in the community repo.
However, some of these things are not directly related to security, such as license&copyright.
Who really owns this? If we move these documents out of the TSC area and into the SSDP-SDLC area, who is in charge of updating this over time? It should still be the TSC, so should these really be moved at all?